centos-pypa-rust/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM quay.io/pypa/manylinux2014_x86_64
+
+WORKDIR /usr/src/app
+
+# download the rustup-init binary and checksum file
+ADD https://static.rust-lang.org/rustup/dist/aarch64-unknown-linux-musl/rustup-init rustup-init
+ADD https://static.rust-lang.org/rustup/dist/aarch64-unknown-linux-musl/rustup-init.sha256 rustup-init.sha256
+
+# validate the checksum of the rustup-init binary
+RUN if [[ $(curl https://static.rust-lang.org/rustup/dist/aarch64-unknown-linux-musl/rustup-init.sha256 | awk '{print $1}') = $(curl -s https://static.rust-lang.org/rustup/dist/aarch64-unknown-linux-musl/rustup-init | sha256sum | awk '{print $1}'
```

Comment:
I pulled the `quay.io/pypa/manylinux2014_x86_64` image locally, and it appears to use glibc, not musl. You can check this by opening up a shell in the image and running `gcc -dM -E - <<< "#include <features.h>" | grep GLIBC`. If you see mentions of glibc, it should be glibc

Reply:
That said, if you switch to aarch64, it is probably musl